logos/logos-ui/src/app/features/billing/billing.html

```
@@ -236,15 +236,11 @@ <h1 class="page-title">Billing</h1>
 
         <div class="btcol-trend" role="cell" [attr.data-label]="vsLabel()">
           @if (team.trendPct !== null) {
-            <span
-              class="trend-badge"
-              [class.trend-badge--up]="team.trendPct > 0"
-              [class.trend-badge--down]="team.trendPct < 0"
-            >
+            <span class="trend-badge">
               {{ team.trendPct > 0 ? '+' : '' }}{{ team.trendPct.toFixed(1) }}%
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)